scripts/Mansion.gd

```diff
@@ -764,6 +817,7 @@ func _on_end_pressed():
 					text2.bbcode_text += person.dictionary("[color=yellow]$name seems no longer to be afraid of you.[/color]\n")
 					person.fear = 0
 			if person.toxicity > 0:
+				var temphealth = person.health #ralphC - bug fix for toxicity incorrectly being blamed for death
```

Comment:
Downstream bugfixes are inappropriate in this context,

Reply:
comment incorporated for next commit